app/helpers/override_range_limit_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Override range limit provided helpers
+module OverrideRangeLimitHelper
+  include RangeLimitHelper
+
+  ##
+  # Overriden helper to add the BH/H and BCE/CE to labels
+  def range_display(solr_field, myparams = params)
+    display = super(solr_field, myparams)
```

Comment:
I _believe_ `super` w/o args passes through the args methods to the super method.

Reply:
Oh good call. I've updated accordingly.